initScripts/x86_64/WindowsServer_2016/Docker_17.06.ps1

```diff
@@ -104,6 +115,7 @@ Function check_docker_opts() {
 
 Function pull_reqProc() {
   Write-Output "Pulling reqProc..."
+  Write-Output "This process might take 10-15 minutes and occupy 15GB of storage space"
```

Comment:
Let's tweak this a little:

The docker pull operation may take up to 15 minutes to complete and take up around 15GB of storage.